<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>368</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>368</width>
    <height>730</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Grayscale Generator</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selectLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Select 3D Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>11</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="resolutionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Output Resolution</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="resolutionSpinBox_x">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>51</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>16384</number>
   </property>
   <property name="value">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="resolutionSpinBox_y">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>51</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>16384</number>
   </property>
   <property name="value">
    <number>512</number>
   </property>
  </widget>
  <widget class="QLabel" name="renderDeviceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Render Device</string>
   </property>
  </widget>
  <widget class="QComboBox" name="renderDeviceComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>91</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>CPU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GPU</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="x_label">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>57</y>
     <width>10</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>678</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ready to Generate.</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="isOutputTXTLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Output TXT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isOutputTXTCheckBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="PushButton" name="generateGrayscalePushButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Grayscale</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="metadataGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>378</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Metadata</string>
   </property>
   <widget class="QLabel" name="verticesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertices =</string>
    </property>
   </widget>
   <widget class="QLabel" name="facesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>   Faces =</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticesText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="facesText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="imageFormatLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Image Format</string>
   </property>
  </widget>
  <widget class="QComboBox" name="imageFormatComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>PNG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JPG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DDS</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="colorDepthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Color Depth</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorDepthComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>8 Bit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16 Bit</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="xFlipLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>X Flip</string>
   </property>
  </widget>
  <widget class="QLabel" name="yFlipLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Y Flip</string>
   </property>
  </widget>
  <widget class="QLabel" name="primeMeridianLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
   <property name="text">
    <string>Prime Meridian</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="xFlipCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="yFlipCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="primeMeridianDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="suffix">
    <string>°</string>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="value">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="deformityCalculator">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>478</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Deformity Calculator</string>
   </property>
   <widget class="QLabel" name="minDistanceLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  Min Distance =</string>
    </property>
   </widget>
   <widget class="QLabel" name="minDistanceText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxDistanceLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  Max Distance =</string>
    </property>
   </widget>
   <widget class="QLabel" name="deformityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>     Deformity =</string>
    </property>
   </widget>
   <widget class="QLabel" name="realRadiusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>   Real Radius =</string>
    </property>
   </widget>
   <widget class="QLabel" name="realDeformity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Real Deformity = </string>
    </property>
   </widget>
   <widget class="QLabel" name="scalingFactor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaling Factor =</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxDistanceText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="deformityText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>75</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="realDeformityText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>125</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="scalingFactorText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="realRadiusDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>98</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000000000000000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="PushButton" name="generateGeopotentialPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Geopotential</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
